extensions/spotify-player/src/api/getUserPlaylists.ts

```diff
@@ -1,14 +1,23 @@
 import { getErrorMessage } from "../helpers/getError";
```

Comment:
Modified to support paginating exposed as an async generator. Used when populating the database.